pkg/scheduler/framework/preemption/preemption_test.go

```diff
@@ -112,6 +112,18 @@ func (pl *FakePreemptionScorePostFilterPlugin) OrderedScoreFuncs(ctx context.Con
 	}
 }
 
+type FakeNodeToStatusReader struct {
+	nodes map[string]*framework.Status
+}
+
+func (f *FakeNodeToStatusReader) Get(nodeName string) *framework.Status {
+	return f.nodes[nodeName]
+}
+
+func (f *FakeNodeToStatusReader) NodesForStatusCode(nodeInfoLister framework.NodeInfoLister, code framework.Code) ([]*framework.NodeInfo, error) {
+	return []*framework.NodeInfo{}, nil
```

Comment:
I don't understand why always empty here? We should test with candidate available as well.